### DISCUSSION TITLE
[mobile web feed] Repo preview's "updated" date is misleading

### DISCUSSION THREAD

**Discussion by mcognetta** [community/community#14469] [Other Feature Feedback, Questions, & Ideas]
On the mobile web (and possibly desktop web) dashboard, the "updated" date on a preview of a repo is misleading. It displays the day of the year, but not the year, so repos appear to have been updated more recently than reality.

A minimal example is shown here, where the updated date implies the repo was last touched a few months ago, but in reality it was ~7 years ago.

![Screenshot_20220407-062957](https://user-images.githubusercontent.com/6856636/162190069-4666b96d-0898-48c8-8989-bfb96ea758c3.png)
![Screenshot_20220407-063036](https://user-images.githubusercontent.com/6856636/162190073-0da5ca39-ac2b-4c1e-988c-76d8e48ddf63.png)

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
